### DISCUSSION TITLE
traveler/14.0.0/iOSVerseMailNotifications

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#96] [Q&A]
# traveler/14.0.0/iOSVerseMailNotifications

Use the Mail Notifications setting to choose when to receive notifications of new mail.

https://help.hcl-software.com/traveler/14.0.0/iOSVerseMailNotifications.html

**Comment by Berokas** [comment]
Settings > Notifications does not exist in the HCL Verse App Version 14.0.9.0
